src/components/VideoCard/VideoCard.test.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import VideoCard from '.';
+import mockedData from '../../mockData/mockData.json';
+
+describe('Video Card', () => {
+  const { items } = mockedData;
+
+  beforeEach(() => {
+    render(<VideoCard snippet={items[1].snippet} />);
+  });
+
+  it('renders provided properties', () => {
+    const card = screen.getByTestId('card');
+    expect(card.children.length).toEqual(2);
+    expect(screen.getByText(items[1].snippet.title)).toBeTruthy();
+    expect(screen.getByText(items[1].snippet.description)).toBeTruthy();
```

Comment:
I agreed with the usage of `data-test-id` here but the expectation statements could be more descriptive using extra vars with the correct names like:
```suggestion
    const videoData = items[1].snippet;
    expect(screen.getByText(videoData.title)).toBeTruthy();
    expect(screen.getByText(videoData.description)).toBeTruthy();
```